providers/signer-connection/test/index.spec.ts

```diff
@@ -1,3 +1,5 @@
+import { describe, it } from "vitest";
+
 // import "mocha";
 // import * as chai from "chai";
 // import { JsonRpcProvider } from "@walletconnect/jsonrpc-provider";
```

Comment:
are these lines supposed to be removed or converted to use `vitest`?

Reply:
These are just commented out signer tests, honestly they won't work anymore anyway, but keeping them in just in case for future reference (these were commented out even before this tooling pr)